storage-operator/go.mod

```diff
@@ -1,35 +1,44 @@
 module github.com/scality/metalk8s/storage-operator
 
+go 1.13
+
 require (
-	github.com/NYTimes/gziphandler v1.0.1 // indirect
 	github.com/go-logr/logr v0.1.0
-	github.com/operator-framework/operator-sdk v0.9.1-0.20190712203509-e1d904fa80a4
+	github.com/operator-framework/operator-sdk v0.15.2
 	github.com/pkg/errors v0.8.1
-	github.com/spf13/pflag v1.0.3
-	github.com/stretchr/testify v1.3.0
-	k8s.io/api v0.0.0-20190612125737-db0771252981
-	k8s.io/apimachinery v0.0.0-20190612125636-6a5db36e93ad
-	k8s.io/client-go v11.0.0+incompatible
-	k8s.io/kube-openapi v0.0.0-20190603182131-db7b694dc208 // indirect
-	sigs.k8s.io/controller-runtime v0.1.12
-	sigs.k8s.io/controller-tools v0.1.10
-)
-
-// Pinned to kubernetes-1.13.4
-replace (
-	k8s.io/api => k8s.io/api v0.0.0-20190222213804-5cb15d344471
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20190228180357-d002e88f6236
-	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190221213512-86fb29eff628
-	k8s.io/client-go => k8s.io/client-go v0.0.0-20190228174230-b40b2a5939e4
+	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.4.0
+	k8s.io/api v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/client-go v12.0.0+incompatible
+	sigs.k8s.io/controller-runtime v0.4.0
 )
 
+// Pinned to kubernetes-1.16.2
```

Comment:
Should this be pinned to different version of k8s depending on version of MetalK8s? Or we don't care?

Reply:
I'm going to close this PR eventually in favor of #2544, but to answer the question: no, it doesn't. This is a *client* version, so if we'd be using APIs/types in *our* code that doesn't work with a deployed version of K8s anyway, we'd be stuck.